From 2e985dfd77fa9147b74f67f61cdf927397bac0e5 Mon Sep 17 00:00:00 2001
From: Lrs121 <lrs@lrsservers.net>
Date: Wed, 29 Jun 2016 12:35:40 -0700
Subject: [PATCH] Makefile patch fixes ASUS_SW_VER error

Used the Makefile listed at
http://forum.xda-developers.com/showpost.php?s=c9ea927a06e33a5d696fd1de7b15f19c&p=65954322&postcount=5

Credits to XDA-USER: T10NAZ

Upstream-Status: Inappropriate
---
 Makefile | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/Makefile b/Makefile
index 0bd2d5de54fb..de5d028396e8 100644
--- a/Makefile
+++ b/Makefile
@@ -671,6 +671,27 @@ ifeq ($(shell $(CONFIG_SHELL) $(srctree)/scripts/gcc-goto.sh $(CC)), y)
 	KBUILD_CFLAGS += -DCC_HAVE_ASM_GOTO
 endif
 
+# +++ ASUS_BSP : miniporting : add ASUS software version support
+ifneq ($(BUILD_NUMBER),)
+        KBUILD_CPPFLAGS += -DASUS_SW_VER=\"$(BUILD_NUMBER)\"
+else
+        KBUILD_CPPFLAGS += -DASUS_SW_VER=\"WI501Q_ENG\"
+endif
+# --- ASUS_BSP : miniporting : jackson : add ASUS software version support
+KBUILD_CPPFLAGS += -DASUS_DEBUG=2
+# +++ASUS_BSP: "support mutliple project build"
+KBUILD_CPPFLAGS += -DASUS_WI501Q_PROJECT=1
+#--- ASUS_BSP: "support mutliple project build"
+
+# +++ ASUS_BSP: "factory compile option support"
+ifneq ($(ASUS_FACTORY_BUILD),)
+        KBUILD_CPPFLAGS += -DASUS_FACTORY_BUILD=1
+endif
+# --- ASUS_BSP: "factory compile option support"
+
+
+
+
 # Add user supplied CPPFLAGS, AFLAGS and CFLAGS as the last assignments
 KBUILD_CPPFLAGS += $(KCPPFLAGS)
 KBUILD_AFLAGS += $(KAFLAGS)
